idea/vara-eth/frontend/src/components/ui/media-query/media-query.tsx

```diff
@@ -0,0 +1,41 @@
+import { PropsWithChildren } from 'react';
+
+import styles from './media-query.module.scss';
+
+function Desktop({ children }: PropsWithChildren) {
+  return <span className={styles.desktop}>{children}</span>;
+}
+
+function Mobile({ children }: PropsWithChildren) {
+  return <span className={styles.mobile}>{children}</span>;
+}
+
+type BreakpointSize = 'Sm' | 'Md' | 'Lg' | 'Xl' | 'Xxl';
```

Comment:
```suggestion
type BreakpointSize = 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
```

Up to you, but seems easier to write.